utils.groovy

```diff
@@ -516,26 +516,32 @@ def build_artifacts(pipecfg, stream, basearch) {
 
 def get_registry_repos(pipecfg, stream, version) {
     def registry_repos = pipecfg.registry_repos ?: [:]
-    // merge top-level registry_repos with stream-specific bits
-    registry_repos += pipecfg.streams[stream].additional_registry_repos ?: [:]
+    // get stream-specific bits additional repos
+    def additional_repos = pipecfg.streams[stream].additional_registry_repos ?: [:]
+    for (repo in ['oscontainer', 'legacy_oscontainer', 'extensions', 'kubevirt']) {
```

Comment:
I think I would prefer it if we didn't hardcode these values here. Can we not just iterate through the keys of `additional_repos`?

Reply:
also suggest to use `repo_id` as the loop var here.

Reply:
> Can we not just iterate through the keys of additional_repos?

I don't think so, because additional_repos and _registry_repos may contain different keys

Reply:
> iterate through the keys of additional_repos

Yeah that make way more sense actually, thanks for the suggestion